peps/pep-3118.rst

```diff
@@ -10,6 +10,18 @@ Created: 28-Aug-2006
 Python-Version: 3.0
 Post-History:
```

Comment:
Shall we add a link to the discussion?

This looks like the relevant thread:

https://mail.python.org/archives/list/python-dev@python.org/thread/X7A5IP4VFJTVCQJSMBXGAH3BVB3A5C3T/#ZTM3PKWBLXD53CAETIEZL2BPWKUG4INM

We can also remove these redundant headers:
```
Version: $Revision$
Last-Modified: $Date$
...
Content-Type: text/x-rst
```